examples/03-Maxwell/Maxwell3D_Segmentation.py

```diff
@@ -0,0 +1,101 @@
+"""
+Maxwell 3D: magnet segmentation
+-------------------------------
+This example shows how you can use PyAEDT to segment magnets of an electric motor.
+The method is valid and usable for any object the user would like to segment.
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+
+from pyaedt import downloads
+from pyaedt import generate_unique_folder_name
+from pyaedt import Maxwell3d
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"`` is needed to generate
+# documentation only.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+
+non_graphical = False
+
+#################################################################################
+# Download .aedt file example
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set local temporary folder to export the .aedt file to.
+temp_folder = generate_unique_folder_name()
+aedt_file = downloads.download_file("object_segmentation", "Motor3D_obj_segments.aedt", temp_folder)
+
+##################################################################################
+# Launch Maxwell 3D
+# ~~~~~~~~~~~~~~~~~
+# Launch Maxwell 3D.
+
+m3d = Maxwell3d(projectname=aedt_file,
+                specified_version="2023.1",
+                new_desktop_session=True,
+                non_graphical=non_graphical)
+
+##################################################################################
+# Create object to access 3D modeler
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create the object ``mod3D`` to access the 3D modeler easily.
+
+mod3D = m3d.modeler
```

Comment:
I would probably name the variable modeler here.
```suggestion
modeler = m3d.modeler
```